src/NServiceBus.AcceptanceTesting/EndpointConfigurationBuilder.cs

```diff
@@ -51,18 +51,16 @@ EndpointCustomizationConfiguration CreateScenario()
             return configuration;
         }
 
-        public EndpointConfigurationBuilder EndpointSetup<T>() where T : IEndpointSetupTemplate, new()
-        {
-            return EndpointSetup<T>(c => { });
-        }
-
-        public EndpointConfigurationBuilder EndpointSetup<T>(Action<EndpointConfiguration> configurationBuilderCustomization) where T : IEndpointSetupTemplate, new()
+        public EndpointConfigurationBuilder EndpointSetup<T>(Action<EndpointConfiguration> configurationBuilderCustomization = null,
+            Action<PublisherMetadata> publisherMetadataAction = null) where T : IEndpointSetupTemplate, new()
```

Comment:
I had to put the lambda here because using the fluent API like `AddMapping` does won't fly since that happens to late, IConfigureEndpointTestExecution has already been called at that stage. Should we go down this path or do we have other ideas?